rego/rego_test.go

```diff
@@ -937,6 +937,27 @@ func TestPrepareAndEvalOriginal(t *testing.T) {
 	assertEval(t, r, "[[2]]")
 }
 
+func TestPrepareAndEvalOnlyOneErrorOccurredPrintOnce(t *testing.T) {
+	module := `
+	package test
+    package test
+	x = input.y
+	`
+
+	r := New(
+		Query("data.test.x"),
+		Module("", module),
+		Package("foo"),
+		Input(map[string]int{"y": 2}),
+	)
+
+	_, err := r.PrepareForEval(context.Background())
+	fmt.Println(err.Error())
```

Comment:
Nit: Can you please remove this print statement. Also let's check that error is non-nil:

```go
if err == nil {
     t.Fatal("Expected error but got nil")
}
```

Reply:
Oops, removed the forgotten print + added non nil assertion